indexer/routing.hpp

```diff
@@ -0,0 +1,141 @@
+#pragma once
+
+#include "coding/bit_streams.hpp"
+#include "coding/elias_coder.hpp"
+#include "coding/reader.hpp"
+#include "coding/varint.hpp"
+#include "coding/write_to_sink.hpp"
+
+#include "base/assert.hpp"
+#include "base/bits.hpp"
+
+#include "std/vector.hpp"
+
+namespace routing
+{
+/// \brief Restriction to modify road graph.
+struct Restriction
+{
+  static uint32_t const kInvalidFeatureId;
+
+  /// \brief Types of road graph restrictions.
+  /// \note Despite the fact more that 10 restriction tags are present in osm all of them
+  /// could be split into two categories.
+  /// * no_left_turn, no_right_turn, no_u_turn and so on go to "No" category.
+  /// * only_left_turn, only_right_turn and so on go to "Only" category.
```

Comment:
Am I right that Restiction(Only, {0, 2, 7}) means that I can only move from link No 0 to link No 2 and then to link No 7?